osu.Game/Rulesets/Edit/EditRulesetContainer.cs

```diff
@@ -68,9 +68,9 @@ internal override DrawableHitObject Add(HitObject hitObject)
             // Process object
             var processor = ruleset.CreateBeatmapProcessor(beatmap);
 
-            processor.PreProcess();
+            processor?.PreProcess();
```

Comment:
how does this null?

Reply:
Mania doesn't have a processor. This is also done in `WorkingBeatmap`.